index.html

```diff
@@ -76,5 +74,168 @@ <h3><a class="connect" href="#">LET'S CONNECT</a></h3>
         </div>
       </div>
     </section>
+
+    <!-- Works Section -->
+    <section class="works">
+      <div class="cards">
+        <div class="project1">
+          <img
+            class="picture"
+            src="Images/Snapshoot Portfolio.png"
+            alt="Snapshoot Portfolio Pic 1"
+          />
+          <h1 class="project-title">Tonic</h1>
+          <ul class="project-summary">
+            <li><h3 class="canopy">CANOPY</h3></li>
+            <li><h3 class="back-end-dev">Back End Dev</h3></li>
+            <li><h3 class="year">2015</h3></li>
+          </ul>
+          <p class="project-details">
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <div class="buttons">
+            <button type="button">html</button></li>
+            <button type="button">css</button>
+            <button type="button">javaScript</button>
+          </div>
```

Comment:
Considering that these technology listings are not buttons but a list, I recommend that you use `li` tags for each of the items and change the div tag to `ul` to enclose the `li` tags.

Remember to adjust this for each of the projects.